### PR TITLE
A few small changes.

### PR DESCRIPTION
**Pull request by achoum**
- Allow for "begin" and "after" operators to work on mix of unix timestamps and non unix timestamps.
- Add tp.duration.to_string to the API (alias for tp.core.data.duration_utils.duration_abbreviation).
- Run black .

### CODE REVIEW

File: docs/public_api_test.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ class PublicAPITest(absltest.TestCase):
     def test_public_symbols(self):
         """Asserts that the symbols exposed under tp.<> are exactly the
         ones we expect."""
-        symbols = {s for s in dir(tp) if not s.startswith("__")}
+        symbols = {s for s in dir(tp) if not s.startswith("__") and s != "api"}
```
Comment (ianspektor) [inline docs/public_api_test.py:73]: `del api` at the end of the root `__init__` is the correct fix for this I believe

**Review by ianspektor — APPROVED** [review]

File: docs/public_api_test.py (hunk 1 above)
Comment (achoum) [reply docs/public_api_test.py]: Good point.
*Removed condition and added "del api".

**Comment by achoum** [comment]
Thanks

**Comment by javiber** [comment]
@achoum I think the docs are failing because you're missing the `__init__.py` inside `temporian/api`

File: temporian/core/operators/filter.py [hunk 2]
```diff
@@ -105,20 +105,7 @@ def before(
     assert isinstance(input, EventSetNode)
 
     if isinstance(timestamp, datetime):
-        if not input.schema.is_unix_timestamp:
```
Comment (javiber) [inline temporian/core/operators/filter.py:108]: Should we have a warning here?

I'm in favor of removing the exception to make the operation more flexible but I'm wondering if we should warn the user about this.

**Comment by achoum** [comment]
> @achoum I think the docs are failing because you're missing the `__init__.py` inside `temporian/api`

Awesome catch. Thanks.

File: temporian/core/operators/filter.py (hunk 2 above)
Comment (achoum) [reply temporian/core/operators/filter.py:108]: Good point.

*Print warning instead of error.
*Add `is_unix_timestamp` setter.

**Pull request merged**